packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/networks.js

```diff
@@ -240,14 +240,55 @@ export const networks = Object.freeze({
     CoinType: CoinTypes.CARDANO,
     Fork: CardanoForks.Haskell,
   }: NetworkRow),
+  CardanoPreviewTestnet: ({
+    NetworkId: 3_00,
```

Comment:
Network `3_00` is the legacy testnet

```suggestion
    NetworkId: 2_40,
```